openscm/core/views.py

```diff
@@ -433,7 +477,7 @@ def length(self) -> int:
         """
         Length of timeseries
         """
-        return self._timeseries_converter.target_length
+        return self._timeseries_converter.target_length  # type: ignore
```

Comment:
Why these `type: ignore`s? What has changed that they are necessary? Maybe use a `cast` instead? Same below...

Reply:
It was the change to returning `ParameterInfo` instead of `TimeseriesView`. When I get rid of that, these will disappear.